src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -45,6 +46,9 @@ void SubgraphBaseTest::run() {
     summary.setDeviceName(targetDevice);
     summary.updateOPsStats(function, status);
 
+    if (targetDevice == "MYRIAD") {
```

Comment:
Why do you make such changes? I strongly disagree

Reply:
Changed.